### PR TITLE
New app example improvements

### PR DESCRIPTION
**Pull request by php-coder**
Improve oc new-app examples:
- remove duplicates
- replace deprecated --image option
- minor improvement

PTAL @bparees

### CODE REVIEW

File: pkg/cmd/cli/cmd/newapp.go [hunk 1]
```diff
@@ -112,9 +109,9 @@ To list all local templates and image streams, use:
 
 To search templates, image streams, and Docker images that match the arguments provided, use:
 
-  %[1]s new-app -S php
```
Comment (bparees) [inline pkg/cmd/cli/cmd/newapp.go]: why did you remove this example?

Comment (php-coder) [inline pkg/cmd/cli/cmd/newapp.go]: The line above talking about searching "templates, image streams, and Docker images" but the example was different and I decided to update example. Do you think I understood this wrong?

Comment (php-coder) [inline pkg/cmd/cli/cmd/newapp.go]: P.S. We can list all 4 possible versions if you think that it will be better.

Comment (bparees) [inline pkg/cmd/cli/cmd/newapp.go]: does `oc new-app -S php` not search all templates/imagestreams/docker images ?

Comment (php-coder) [inline pkg/cmd/cli/cmd/newapp.go]: Yes, it does.

I've pushed updated example then.

**Comment by bparees** [comment]
squash and lgtm.

**Comment by php-coder** [comment]
@bparees Ready for merge.

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
@php-coder needs some doc regen.  make update.

**Comment by bparees** [comment]
This will have to be post 3.3 at this point.

**Comment by php-coder** [comment]
[test] again; before it failed because of https://github.com/openshift/origin/issues/10566

**Comment by php-coder** [comment]
@bparees Could you merge it?

**Comment by bparees** [comment]
@php-coder was going to wait for post 3.3 but since we've extended the release [merge]
